### PR TITLE
feat(integration): Add Webpack zero config integration test with Karma

### PR DESCRIPTION
**Pull request by Archcry**
Add integration test for Webpack zero config with Karma.

Depends on https://github.com/stryker-mutator/stryker/pull/678, build will be red untill https://github.com/stryker-mutator/stryker/pull/678 is merged into master and master is merged here.

### CODE REVIEW

File: integrationTest/test/webpack-zero-conf-karma/.gitignore [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```
Comment (nicojs) [inline integrationTest/test/webpack-zero-conf-karma/.gitignore]: Is this `.gitignore` file really needed? We have a pretty good gitignore file at the root, no?

Comment (Archcry) [reply integrationTest/test/webpack-zero-conf-karma/.gitignore]: Does it filter the dist folder?

**Comment by Archcry** [comment]
Tests fail because the port for Karma is already in use, Node LTS and 4 don't have this problem it seems.

**Comment by nicojs** [comment]
Node 4 doesn't have the problem because the angular project tests are not run on node 4. Because angular-cli requires node 6 and up.

**Comment by Archcry** [comment]
It's not because of angular-cli, it's because karma tries to start a server on a port that is already occupied.

**Comment by nicojs** [comment]
I've resolved the port conflict in the integration tests. Will be merged when the build is green.
